packages/asc-ui/src/utils/themeUtils.ts

```diff
@@ -152,6 +152,22 @@ export const focusStyleText = () => ({
   }
 `
 
+export enum FocusStyleEnum {
+  outline,
+  fill,
+  none,
+}
+
+export const getFocusStyle = (focusStyle: string = 'fill') => {
```

Comment:
perhaps a little comment / documentation on how this exactly works? Took me a little while to understand what it does

Reply:
will do